lib/card.rb

```diff
@@ -14,11 +14,9 @@ class Card < ActiveRecord::Base
 
   has_many :references_from, :class_name => :Reference, :foreign_key => :referee_id
   has_many :references_to,   :class_name => :Reference, :foreign_key => :referer_id
-  has_many :acts, :order => :id
-  has_many :actions, :order => :id, :conditions=>{:draft => [nil,false]}
-  has_many :drafts, :order=>:id, :conditions=>{:draft=>true}, :class_name=> :Action
-
-  cache_attributes 'name', 'type_id' # review - still worth it in Rails 3?
+  has_many :acts, -> { order :id }
+  has_many :actions, -> { where( :draft=>[nil,false]).order :id }
+  has_many :drafts, { :class_name=> :Action }, -> { where( :draft=>true ).order :id }
```

Comment:
ok, great, it sounds like we're on the same page.  I do think we should try to get to >3.2 or >3.2.x if there are problems with the early versions.  

This probably means we need a separate semaphore setup for testing whatever we decide is the lowest ruby version and the lowest rails version we support.

By the way, I think wagn is currently broken in ruby 2.2.0 (but working well in 2.1.5)